services/ansible/ansible.tester.js

```diff
@@ -4,6 +4,8 @@ const Joi = require('joi')
 const ServiceTester = require('../service-tester')
 const { isMetric } = require('../test-validators')
 
+const isQualityScore = Joi.string().regex(/^[0-9]+\.[0-9]/)
```

Comment:
Any reason we're validating the input with `number()` but then using a regex in the test?